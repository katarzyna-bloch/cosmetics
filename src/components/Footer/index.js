import styled from 'styled-components'

import media from '../../utils/media'

const Wrapper = styled.footer`
  background-color: lavender;
  padding: 12px;
  text-align: center;
`

const Link = styled.a`
  text-decoration: none;
  color: black;
  margin: 30px;

  ${media.phone} {
    margin: 10px;
  }
`

const Footer = () => (     
  <Wrapper>
    <Link href="http://facebook.com" target="_blank">Facebook</Link>
    <Link href="http://instagram.com" target="_blank">Instagram</Link>
    <Link href="http://youtube.com" target="_blank">YouTube</Link>
  </Wrapper>
)

export default Footer
