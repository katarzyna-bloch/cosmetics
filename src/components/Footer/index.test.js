import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import Footer from '.'

it('should render correctly', () => {
  let wrapped = shallow(<Footer />)
  expect(toJson(wrapped)).toMatchSnapshot()
})
