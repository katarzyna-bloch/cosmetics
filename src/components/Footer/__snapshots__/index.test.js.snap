// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render correctly 1`] = `
<styled.footer>
  <styled.a
    href="http://facebook.com"
    target="_blank"
  >
    Facebook
  </styled.a>
  <styled.a
    href="http://instagram.com"
    target="_blank"
  >
    Instagram
  </styled.a>
  <styled.a
    href="http://youtube.com"
    target="_blank"
  >
    YouTube
  </styled.a>
</styled.footer>
`;
